azure-pipelines/signing.yml

```diff
@@ -429,6 +429,11 @@ jobs:
       inputs:
         command: custom
         arguments: 'pack $(Build.ArtifactStagingDirectory)/vs-insertion/staging/vcpkg.nuspec -NoDefaultExcludes -OutputDirectory "$(Build.ArtifactStagingDirectory)/vs-insertion/drop" -Properties version=$(VCPKG_FULL_VERSION)'
+    - task: MSBuild@1
+      displayName: 'Sign VS Insertion NuGet Package'
+      inputs:
+        solution: 'azure-pipelines\nuget-package.signproj'
+        msbuildArguments: '/p:OutDir=$(Build.ArtifactStagingDirectory)\vs-insertion\drop /p:IntermediateOutputPath=$(Build.ArtifactStagingDirectory)\vs-insertion\drop'

```

Comment:
Not sure if this is a better name
```suggestion
        msbuildArguments: '/p:OutDir=$(Build.ArtifactStagingDirectory)\vs-insertion\drop /p:NugetPkgPath=$(Build.ArtifactStagingDirectory)\vs-insertion\drop'
```

Reply:
It must be `IntermediateOutputPath`. See https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_wiki/wikis/DevDiv.wiki/650/MicroBuild-Signing

<img width="515" alt="image" src="https://github.com/microsoft/vcpkg-tool/assets/1544943/2a066b85-8741-4b1f-9c9c-cb3b4d622f12">